be/src/olap/rowset/segment_v2/rle_page.h

```diff
@@ -129,7 +124,26 @@ class RlePageBuilder : public PageBuilder {
         (void)ret;
     }
 
+    Slice finish_and_release() override {
+        return _finish(RELEASE);
+    }
+
 private:
+
+    Slice _finish(BuilderReturnType builderReturnType) {
```

Comment:
```suggestion
    Slice _finish(BuilderReturnType type) {
```
In c++, we use lowercase and underline for variable names

Reply:
👌